### PR TITLE
docs: update README, add LICENSE

### PR DESCRIPTION
**Pull request by secretfader**
Update the Readme to include a CI badge (now that it is passing), and add a License document (GNU GPL 3.0, same as the Chain) to bolster the legal framework for this repo.

### CODE REVIEW

**Review by ETeissonniere — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -14,3 +16,7 @@ Change directory to the resepctive mission (e.g. cd simple-mission) and then
     cargo test
     cargo contract test
     cargo contract build
+
+## License
+
+Copyright 2023 Intergalatic Labs, Inc. Released under the GNU General Public License, version 3.0. See `LICENSE` for further details.
```
Comment (simonsso) [inline README.md:22]: @secretfader 
Is this really the name of our legal entity or did we copy the from from somewhere?

Comment (secretfader) [reply README.md:22]: Nope, this changed a few weeks back. This is now our US business entity, which holds most copyrights.

Comment (aliXsed) [reply README.md:22]: Is it Intergalactic or Intergalatic?

Comment (secretfader) [reply README.md:22]: "Intergalactic." Fixed in #23.

**Pull request merged**